src/lib/util.js

```
@@ -6,7 +6,5 @@ exports.getQuote = function * (connector, query) {
   try {
     const res = yield request.get(connector + '/quote').query(query)
     return res.body
-  } catch (err) {
-    if (!err.response || err.response.body.id !== 'AssetsNotTradedError') throw err
-  }
+  } catch (err) { }
```

Why do you swallow the error here?